content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -683,6 +687,62 @@ More configuration options can be found on the [kong-plugin-ddtrace][3] plugin d
 [3]: https://github.com/DataDog/kong-plugin-ddtrace#configuration
 
 {{% /tab %}}
+
+{{% tab "Apache HTTPd" %}}
+
+## HTTPd with Datadog module
+Datadog provides a [module][1] for enhancing [Apache HTTP Server][2] capabilities with APM Tracing.
+
+### Module installation
+<div class="alert alert-warning">
+  <strong>Note:</strong> Only Apache HTTP Server 2.4.x is supported.
+</div>
+
+The module is provided as a shared library for dynamical loading by HTTPd. Each supported platform
+and architecture has its own artifact hosted on [httpd-datadog's repository][1].
+
+Run the following script to download the latest version of the module:
+
+```bash
+get_latest_release() {
+  curl --silent "https://api.github.com/repos/$1/releases/latest" | jq --raw-output .tag_name
+}
+
+RELEASE_TAG=$(get_latest_release DataDog/httpd-datadog)
+wget "https://github.com/DataDog/httpd-datadog/releases/download/$RELEASE_TAG/linux-x86_64-mod_datadog.tar.gz"
```

Comment:
It is not guaranteed to have `curl` and `wget` installed. However, it is way more probable to have `curl` installed due to `libcurl` being basically ubiquitous.

Since you are already using `curl` above in the function, I do not see any additional benefit in using `wget` here, and I would suggest to replace it with `curl -L -o linux-x86_64-mod_datadog.tar.gz <url>`

Reply:
Agree.